modules/emx_digitalBidAdapter.js

```diff
@@ -151,40 +181,45 @@ export const spec = {
     return true;
   },
   buildRequests: function (validBidRequests, bidderRequest) {
-    const page = bidderRequest.refererInfo.referer;
-    let emxImps = [];
-    const timeout = config.getConfig('bidderTimeout');
+    const emxImps = [];
+    const timeout = bidderRequest.timeout || "";
```

Comment:
Should use single-quotes instead double-quotes.  This will fix the lint error.

Reply:
Done -- thank you